.ci/apm-beats-update.groovy

```diff
@@ -26,7 +26,7 @@ pipeline {
     quietPeriod(10)
   }
   triggers {
-    issueCommentTrigger('(?i).*/run\\s+(?:apm-beats-update\\W+)?.*')
+    issueCommentTrigger('(?i)^/run\\s+(?:apm-beats-update\\W+)?.*')
```

Comment:
Is this change related somehow to the migration to `obltGitHubComments` or is it a separate fix? It seems as if it's separate and so perhaps we should update the description of this PR?

Reply:
Unrelated but forcing to use `/run` rather than `bla bla bla /run`. So I'll revert it to keep the PR clean

Reply:
```suggestion
    issueCommentTrigger('(?i).*/run\\s+(?:apm-beats-update\\W+)?.*')
```